resources/tests.mjs

```diff
@@ -332,6 +332,53 @@ Suites.push({
     ],
 });
 
+Suites.push({
+    name: "NewsSite-Next",
+    url: "tentative/newssite/news-next/dist/index.html",
+    async prepare(page) {
+        await page.waitForElement("#close-toast-link");
+    },
+    tests: [
+        new BenchmarkTestStep("NavigateToUS", (page) => {
+            page.querySelector("#close-toast-link").click();
+            page.querySelector("#navbar-dropdown-toggle").click();
+            page.querySelector("#navbar-dropdown-toggle").click();
+            page.querySelector("#navbar-navlist-us-link").click();
+        }),
+        new BenchmarkTestStep("NavigateToWorld", (page) => {
+            page.querySelector("#close-message-link").click();
```

Comment:
Removing this line makes this test consistently pass in Safari so maybe we can do that for now?
We can always revisit & debug this in more detail but it seems like landing the rest of content into the tentative directory seems quite useful.

Reply:
done - thanks for finding!